sql-plugin/src/main/scala/org/apache/spark/sql/rapids/ExternalSource.scala

```diff
@@ -31,24 +31,19 @@ import org.apache.spark.sql.v2.avro.AvroScan
 import org.apache.spark.util.{SerializableConfiguration, Utils}
 
 object ExternalSource extends Logging {
+  val avroScanClassName = "org.apache.spark.sql.v2.avro.AvroScan"
```

Comment:
nit: :s/avroScanClassName/AVRO_SCAN_CLASS/g

Reply:
this is a suggestion for your PR NVIDIA/spark-rapids#5716, you can change in your PR branch addressing more review comments